app/controllers/statements_controller.rb

```diff
@@ -13,22 +13,18 @@ def initialize
     super
   end
 
-  # GET /accounts/:account_id/facilities/:facility_id/statements/:id
+  # GET /accounts/:id/statements
+  def index
+    @order_details = @account.order_details.has_statement.by_statemented_at
```

Comment:
I think we're looking for a list of statements, not a list of order details. We don't need to see the individual order details in this view.